frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -642,13 +640,18 @@ const TestResultsList = () => {
   };
 
   const filter = (params: FilterParams) => {
+    const searchParams = {
+      facility: activeFacilityId,
+      ...filterParams,
+      ...params,
+    };
+    const nonNullSearchParams = Object.fromEntries(
+      Object.entries(searchParams).filter(([_, v]) => v !== null)
+    ) as Record<string, string>;
```

Comment:
since some of the searchParams were optional and URLSearch params wants a `Record<string, string>`, TS was complaining about null checks. Adding this here to fix